FactoryWeather/App/VIP Settings/SettingsRouter.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SettingsRouter.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 09/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+protocol SettingsRoutingLogic {
+    func getNewWeather(selectedLocation: Location?)
+    func dismissSettingsScene()
```

Comment:
The `SettingsScene` part of `dismissSettingsScene` is redundant. Usually, I use `func unwindBack()` as the default method naming for `router` methods that `dismiss` the `router`'s scene, but we could settle on anything, as long as we always use that particular naming.

Reply:
Ok, will be using `unwindBack`.